RevenueCatUI/CustomerCenter/View+PresentCustomerCenter.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  View+PresentCustomerCenter.swift
+//
+//  Created by Toni Rico Diez on 2024-07-15.
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+/// Presentation options to use with the [presentCustomerCenter](x-source-tag://presentCustomerCenter) View modifiers.
+public enum CustomerCenterPresentationMode {
+
+    /// Customer center presented using SwiftUI's `.sheet`.
+    case sheet
+
+    /// Customer center presented using SwiftUI's `.fullScreenCover`.
+    case fullScreen
+
+}
+
+extension CustomerCenterPresentationMode {
+
+    // swiftlint:disable:next missing_docs
+    public static let `default`: Self = .sheet
+
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable, message: "RevenueCatUI does not support macOS yet")
+@available(tvOS, unavailable, message: "RevenueCatUI does not support tvOS yet")
+@available(watchOS, unavailable, message: "CustomerCenterView does not support watchOS yet")
+@available(visionOS, unavailable, message: "CustomerCenterView does not support visionOS yet")
+extension View {
+
+    /// Presents the ``CustomerCenter``.
+    /// Example:
+    /// ```swift
+    /// var body: some View {
+    ///    YourApp()
+    ///      .presentCustomerCenter()
+    /// }
+    /// ```
+
```

Comment:
Is this missing a `///` or will this block be associated with `presentCustomerCenter(` just fine?

Reply:
Ohh that was unintended. Didn't check, but I would think it could bring issues. Will fix. Nice catch!